### PR TITLE
homepg screen

### PR DESCRIPTION
**Pull request by aurobindoGupta**
image

### CODE REVIEW

**Review by jeetbhalani15 — APPROVED** [review]
looks ok

File: index.html [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+      <!-- ................PAGE CONTENT ............................... -->
+
+      <div class="pageContent">
+        <div class="top-preview">
+          <div class="preview-card style1">
+            <div class="style1-group">
+              <img
+                class="preview-card-image"
+                src="/utils/images/productRectangle1.png"
+              />
```
Comment (Naveenchandar) [inline index.html]: Please provide alt description for accessibility

File: utils/colors.css [hunk 2]
```diff
@@ -0,0 +1,9 @@
+:root{
+    --background-color: #E5E5E5;
+    --navBar-color:#FFFFFF;
+    --primary-color:#C4C4C4;
+    --btn-primary-color:#9A9A9A;
+    --btn-secondary-color:#FFFFFF;
+    --btn-focus-color:#3B82F6;
+    --stdWhite-color:#FFFFFF;
+}
```
Comment (Naveenchandar) [inline utils/colors.css]: #FFFFFF is repeated. You can remove the repeated ones.

File: index.html (hunk 1 above)
Comment (aurobindoGupta) [reply index.html]: done

File: utils/colors.css (hunk 2 above)
Comment (aurobindoGupta) [reply utils/colors.css]: done

**Review by Naveenchandar — APPROVED** [review]
Looks good

**Pull request merged**